stories/blocks/header-block.stories.js

```diff
@@ -0,0 +1,73 @@
+import { storiesOf } from '@storybook/html';
+
+import '../../resources/assets/styles/main.scss';
+import boat from  '../../resources/assets/images/boat.png';
+
+storiesOf('Elementy/header', module)
+    .add('Header', () => `
+    <h2 class="subtitle">
+      Header
+    </h2>
+    <hr />
+    <div class="header">
+      <h2 class="header__title title text--center">
+        Doskonała lokalizacja
+      </h2>
+      <h3 class="header__sub subtitle text--center">
+        Wysoki standard
+      </h3>
```

Comment:
To wszystko powinno być zamknięte w 1 nagłówku 
`<h2> span.header__title i span.header__sub</h2>`